Content.Server/GameObjects/Components/Kitchen/KitchenMicrowaveComponent.cs

```diff
@@ -403,5 +407,27 @@ private void ClickSound()
 
         }
 
+        public SuicideKind Suicide(IEntity victim, IChatManager chat)
+        {
+            victim.TryGetComponent<BodyManagerComponent>(out var bodyManagerComponent);
+            if (bodyManagerComponent != null)
+            {
+                var heads = bodyManagerComponent.GetBodyPartsOfType(BodyPartType.Head);
+                foreach (var head in heads)
+                {
+                    var droppedHead = bodyManagerComponent.DisconnectBodyPart(head, true);
+                    _storage.Insert(droppedHead);   
+                }
+                chat.EntityMe(victim, Loc.GetString("is trying to cook {0:their} " + (heads.Count() > 1 ? "heads!": "head!"), victim));
```

Comment:
Note that this will generate two lines to translate: one with "head!" and one with "heads!".
It is not bad since the rest of the sentence my be accorded with the number of head. However, you can try to use the plural feature of the localization library. I am not sure about the correct order of argument though:

```c#
Loc.GetStringPlural("is trying to cook {0:their} head!", "is trying to cook {0:their} heads!", head.Count(), victim);
```